openmmtools/multistate/multistatesampler.py

```diff
@@ -1014,7 +1013,8 @@ def _read_options(check_iteration):
                                                     "Either your data is fully corrupted or something has gone very "
                                                     "wrong to see this message. "
                                                     "Please open an issue on the GitHub issue tracker if you see this!")
-            except:
+            except Exception as err:
+                raise err
```

Comment:
We probably want to change this to either not raise the exception and catch whatever exception it can raise, or catch a specific exception. This doesn't seem to affect the tests but maybe we do affect applications/users by raising the exception here.

Reply:
Since I didn't understand what kind of error this could throw, I was thinking we would need to catch this one in the wild and I rather do that than not know. We could run perses on this PR and see if it causes a probme.

Reply:
I see, okay. I guess that should work.